man/shard.yml.5

```diff
@@ -582,6 +582,24 @@ Extends the \fIgit\fP resolver, and acts exactly like it.
 \fBExample:\fP \fIbitbucket: tom/library\fP
 .RE
 .sp
+\fBhg\fP
+.RS 4
+A Mercurial repository URL (string).
+.sp
+The URL may be \c
+.URL "https://www.mercurial\-scm.org/repo/hg/help/clone" "any protocol"
+supported by Mercurial, which includes SSH and HTTPS.
+.sp
+The Merurial repository will be cloned, the list of versions (and associated
```

Comment:
```suggestion
The Mercurial repository will be cloned, the list of versions (and associated
```

Reply:
This change needs to go into the source file, but I'll append it.